src/comp/comp.py

```diff
@@ -24,48 +24,48 @@ def __repr__(self):
 # Write a list comprehension that creates a list of names of everyone
 # whose name starts with 'D':
 print("Starts with D:")
-a = []
+a = [first_name.name for first_name in humans if first_name.name[0] == 'D']
 print(a)
 
 # Write a list comprehension that creates a list of names of everyone
 # whose name ends in "e".
 print("Ends with e:")
-b = []
+b = [last_name.name for last_name in humans if last_name.name[-1] == 'e']
 print(b)
 
 # Write a list comprehension that creates a list of names of everyone
 # whose name starts with any letter between 'C' and 'G' inclusive.
 print("Starts between C and G, inclusive:")
-c = []
+c = [first_name.name for first_name in humans if first_name.name[0] in ['C', 'D', 'E', 'F', 'G']]
```

Comment:
This is perfect, just throwing it in here also that you can iterate through a string in python:

```
if first_name.name[0] in "CDEFG"
```

Just a little trick that may come in handy in the future.